lib/Views/global.scss

```diff
@@ -1,5 +1,5 @@
-@import "~terriajs-variables";
-@import "~terriajs/lib/Sass/global/global";
+@use "../Styles/variables.scss";
+@use "~terriajs/lib/Sass/global/global";
```

Comment:
@na9da The 0.3.0 release notes state the `~terriajs` alias was replaced with relative paths. Is it the release note or this usage that is wrong?

There is also a usage of `~terriajs` left in terriajs (https://github.com/TerriaJS/terriajs/pull/7427).